man/readtags.1.rst.in

```diff
@@ -92,6 +92,13 @@ human-readable, but when utilizing readtags output in a script or a client
 tool, ``-E`` option should be used. See ctags-client-tools(7) for more
 discussion on this.
 
+About printing input fields ({tagfile} in tags(5)) with ``-E`` option:
+readtags prints the input field literally (as it is in the tags file)
+when ``TAG_OUTPUT_MODE`` pseudo tag has "u-ctags" and
+``TAG_OUTPUT_FILESEP`` has "slash" as values for their input fields.
+If this condition is not satisfied, readtags prints the fields with
+unescaping. About the pseudo tags, see ctags-client-tools(7).
```

Comment:
I think I understand most of the things now, but this is still a bit confusing: If the condition is not satisfied, then according to tags(5), the input field won't contain any escape sequences, right? then we shouldn't need to "print with unescaping" as there are no escape sequences to unescape.

Reply:
You are correct. This should be 
```
If this condition is not satisfied, readtags prints the fields literally (as it is in the tags file) regardless ``-E`` option is given or not.
```

Surprisingly, readtags built with this pull request doesn't behave as I wrote here.
I must update this pull request carefully.

Reply:
Again, You are correct. After drawing the figure, I understand readtags print the input fields as-is when -E option is given. It doesn't depend on the pseudo tags.
However, the behavior of libreadtags depends on the pseudo tags.